deals/deals.go

```diff
@@ -19,12 +20,14 @@ import (
 	"github.com/filecoin-project/specs-actors/actors/abi"
 	"github.com/filecoin-project/specs-actors/actors/abi/big"
 	"github.com/ipfs/go-cid"
+	"github.com/ipfs/go-datastore"
 	logging "github.com/ipfs/go-log/v2"
 	"github.com/textileio/powergate/util"
 )
 
 const (
 	chanWriteTimeout = time.Second
+	dealTimeout      = time.Hour * 24
```

Comment:
At some point we have to give up monitoring an in process deal. I just chose 24 hours.

Reply:
Yeah, I think is is a safe reasonable value.